Advanced/One-Click Macros/Measure in Calc Item.csx

```diff
@@ -0,0 +1,216 @@
+#r "Microsoft.VisualBasic"
+using Microsoft.VisualBasic;
+
+// CHANGE LOG: 
+// '2021-10-20 / B.Agullo / 
+// '2021-11-22 / B.Agullo / Totally rewrote the script as did not work the way it was 
+
+
+// Instructions:
+// select the calculation items and and the measure(s) for which the calculation should be shown. 
+// the first time you will be asked to introduce a name for your calc group and dummy measure 
+// second time on it will reuse the same group and dummy measure to add new calc items 
+// the script will create a new calc item for eachs elected measure
+// A pop-up will show if the measure has been previously selected and thus already has a calculation item with its name
+
+
+// TODO: SWAP PRECEDENCE OF NEW CALC GROUP AND SELECTED CALC GROUP 
+
+//
+// ----- do not modify script below this line -----
+//
+
+string affectedMeasures = ""; 
+
+string selectedCalcItems = ""; 
+string selectedCalcItemsCalcGroupName = ""; 
+
+CalculationGroup selectedCalculationGroup = null as CalculationGroup; 
+Column selectedCalcItemsCalcGroupColumn = null as Column; 
+
+
+
+if (Selected.Measures.Count == 0) {
+    
+    Error("No measures selected"); 
+    return; 
+
+} else if (Selected.CalculationItems.Count == 0) { 
+
+    Error("No calculation items selected"); 
+    return; 
+
+} else {
+    
+    //foreach(var m in Selected.Measures) {
+    //    if(affectedMeasures == "") {
+    //        affectedMeasures =  "[" + m.Name + "]";
+    //    } else {
+    //        affectedMeasures = affectedMeasures + ",[" + m.Name + "]" ;
+    //    };
+    //}; 
+
+    
+    // create in-line table with selected calc item names 
+    foreach(var ci in Selected.CalculationItems) { 
+        
+        if(selectedCalcItems == "") {
+            selectedCalcItems += "{\"" + ci.Name + "\""; 
+
+        } else { 
+            selectedCalcItems += ",\"" + ci.Name + "\""; 
+
+        }; 
+        
+        if(selectedCalcItemsCalcGroupName == "") {
+            
+            
+            //selectedCalculationGroup = ci.CalculationGroupTable;
```

Comment:
Commented-out code in production scripts should be:

A. For example/instruction/documentation purposes
B. Optional functionality, disabled by default, which the user can choose to enable by un-commenting it
C. Removed

If this commented-out code block fits the A and/or B criteria above, please add a new comment on the line above it explaining its purpose, and how to use it (if applicable). Otherwise, please remove it. Thanks!